File: class.cpp
#include <iostream>
#include <string>
#include <cstring>
#include <algorithm>
using namespace std;
// This is the first project in OOP
class car
{
    // In private scope you can put Object's attributes
private:
    char name[20];  // Name Attribute
    char color[20]; // Color Attribute
    int maxspeed;   // Maxspeed Attribute
    int model;      // Model Attribute
    // In public scope you can put funcions
public:
    // Funcions to Set Values
    void setName(char n[])
    {
        strcpy(name, n);
    }
    void setcolor(char c[])
    {
        strcpy(color, c);
    }
    void setMaxspeed(int s)
    {
        maxspeed = s;
    }
    void setModel(int m)
    {
        model = m;
    }
    // Funcions to Get Values
    char *getName() // I put * to return array of character
    {
        return name;
    }
    char *getColor()
    {
        return color;
    }
    int getMaxspeed()
    {
        return maxspeed;
    }
    int getModel()
    {
        return model;
    }
    // This funcion print all attribute of object
    void print()
    {
        cout << "Name = " << name << "\n"
             << "Color = " << color << "\n"
             << "Maxspeed = " << maxspeed << "\n"
             << "Model = " << model << "\n";
    }
};
int main()
{
    // I created object from class "car"
    car x;
    // you can just use Member funcion just from the Object
    x.setName("Kia");
    x.setcolor("Red");
    x.setMaxspeed(300);
    x.setModel(2016);
    x.print();
}

File: class.py
#This is the Frist project of OOP
#python is multiparadigm programming language
class car:
# when we create a class python looks for __init__ to verify data to the object
    def __init__(self,name,color,maxspeed,model): # Here you can write the parameter of the Object
        self.__name=name # name attribute =name as parameter 
        self.__color=color # color attribute =color as parameter 
        self.__maxspeed=maxspeed # maxspeed attribute =maxspeed as parameter 
        self.__model=model # moded attribute =model as parameter 
    #here set function to change the private attribute
    # for your information (private attribute in python is just a name convension) 
    def setName(self,new_name):
        self.__name=new_name
    def setcolor(self,new_color):
        self.__color=new_color
    def setMaxspeed(self,new_Maxspeed):
        self.__maxspeed=new_Maxspeed
    def setmode(self,new_model):
        self.__model=new_model
    #Funcion to get private attributes 
    def getName(self):
        return self.__name
    def getcolor(self):
        return self.__color
    def getmaxspeed(self):
        return self.__maxspeed
    def getmode(self):
        return self.__model
    #printall is a function to print every thing
    def printall(self):
        print(f"Name = {self.__name} \nthe color = {self.__color} \nthe maxspeed = {self.getmaxspeed()} \nthe model = {self.__model} ")
# here we made class instantiate means create a class from the object 
car1=car("Kia","Red",203,2016)
car1.setName("BMW")
car1.printall()
        